### DISCUSSION TITLE
Dev v4.2

### DISCUSSION THREAD

**Discussion by Floppe** [Blackymas/NSPanel_HA_Blueprint#1456] [General]
Hi,

Tested current dev branch and some thoughts. 
- Chips icons does not override with selected icons anymore, but a lightbulb or a switch/door is shown now instead.
- Hardware button indicators does not always show correct state if they are triggered from HA or webserver while screen is on.
- Climate heating broke for me, as the setpoint is not the same as target point for my geothermal heater. Perhaps a user setting in ESP? When I want to increase one step from 19 to 20 degrees then it is set to 40-50 degrees which is the target heating water temp. and all goes haywire in the basement and extremely hot in the house :)
- Entities page does not show labels when I have custom names. #1455
- Perhaps we could have a default icon based on UOM if none is entered/overiden for entities? Basics like climate, humidity, power?

**Comment by edwardtfn** [comment]
Thanks a lot for testing this... I will split into different issues and work on each of those.

**Reply by edwardtfn** [reply]
There is:
- #1457 
- #1458 
- #1459 
- #1460 
- #1461 
- #1462 
- #1463

Those are all in the milestone for v4.2: https://github.com/Blackymas/NSPanel_HA_Blueprint/milestone/2

**Reply by Floppe** [reply]
Great. Tell me to stop if I test too much :)

- Alarm page does not update selected mode when changing, but the other panels catches is ASAP on both home & alarm screen 👍 
- This perhaps not an issue, but also not possible to switch from away to home. Need to disarm first before it accepts a new alarm state. The other way seems to work, home -> away which perhaps is more used.

**Reply by edwardtfn** [reply]
> Tell me to stop if I test too much :)

Please don't stop. 😄 
It's better to find things now than after the release.
Thanks a lot for testing and reporting all those things.

I couldn't test myself for a few days, so I was expecting many bugs.

**Reply by edwardtfn** [reply]
By the way, when you say "other panels", which version are you running on those?

**Reply by Floppe** [reply]
It's an older dev version where I have my energy page on. From 11 december, 671b1127ec69497c9897f48d545ddad3e26b4540.

**Reply by edwardtfn** [reply]
Nice to hear you have an "Energy page". Is that something you changed in the HMI?
Have you seen #168 ?

**Reply by Floppe** [reply]
Yes. At first I did have it configurable if you remember, I asked for bigger font icons. Later I created a static background and labels so it fits my needs. Also so both daily and current consumption fits on the same page.

So I think my version is nothing for the public mass. https://github.com/Blackymas/NSPanel_HA_Blueprint/discussions/838#discussioncomment-7530806

**Reply by edwardtfn** [reply]
That's right! I remember that.
As it was in "Discussions" and with nothing related to the energy dashboard in the name I've missed that.
I wanna work in a "Power page" for v4.3. 😉

**Comment by edwardtfn** [comment]
I've just pushed a bunch of changes to `dev`. I would be nice if you can give me some feedback. I wanna go to `beta` tomorrow and if everything goes right I wanna release on the 3rd of January.

**Comment by Floppe** [comment]
I have short on time, but a fast test and seems to work quite ok. I think there are some weirdness with the chips. Chip 4 is blue for me now. Chip 3 perhaps wrong.. seems to trigger for chip 1 instead.

Entities, QR and Alarm panel okay.

How about the climate page. I did see a commit referencing it (and dual state), but how if possible should I set correct target temp sensor? Now it uses 48.8 for heat when I would want 19. (We have -28 Celsius outside so that's why radiator water is heating to 49)

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/238729/15c69c55-a264-42b1-a76d-f7dfc59909e9)

Later I would want to try dual state also for tap water.

**Reply by edwardtfn** [reply]
Could you please share the attributes of these climate entities? From **Developer tools**, oj the tab *States***.
And thanks a lot for your tests. I will take a look at the chips, but I haven't changed anything there since a couple of weeks. The one on blue is a light?

**Reply by Floppe** [reply]
Yes, a led strip. 
I think the chip triggers have been a little off for a longer time, but correct chip have lit up.

climate.heat

> hvac_modes: off, auto
min_temp: 10
max_temp: 40
target_temp_step: 1
current_temperature: 48.6
temperature: 19
hvac_action: heating
friendly_name: Heat
supported_features: 1

**Reply by edwardtfn** [reply]
- #1459
